network-api/networkapi/buyersguide/templates/admin/index_view.html

```diff
@@ -3,7 +3,7 @@
 {% block header_extra %}
   {{ block.super }}
   <div class="right">
-    <form action="/privacynotincluded/refresh-cache" method="post">
+    <form action="/en/privacynotincluded/refresh-cache" method="post">
```

Comment:
This PR will need to be rebased, and then this line change can be removed, as it was fixed elsewhere.